internal/connect/command_custom_plugin_list.go

```diff
@@ -22,28 +23,35 @@ func (c *customPluginCommand) newListCommand() *cobra.Command {
 		Example: examples.BuildExampleString(
 			examples.Example{
 				Text: "List custom connector plugins in the org",
-				Code: "confluent connect custom-plugin list",
+				Code: "confluent connect custom-plugin list --cloud aws",
 			},
 		),
 	}
 
+	cmd.Flags().String("cloud", "", "Filter plugins by cloud provider.")
```

Comment:
Can you add autocompletion? There is already a function you might be interested in using called `pcmd.AddCloudFlag()`